server/env.js

```diff
@@ -12,7 +12,7 @@ const {
 // ONLY CHANGE THESE VARIABLES
 // (or those in dotenv, which require restart)
 const OPEN_ALL_CONTESTS = true;
-const HIDE_FIREBASE = false;
+const HIDE_FIREBASE = true;
```

Comment:
What does `HIDE_FIREBASE` do again?

Reply:
Since I do not have access to the `FIREBASE_PRIVATE_KEY`, if I do not turn off `admin.initializeApp` in `firebase.js` the app won't work.

I've changed it now to be:
`const IS_FIREBASE_OFF = IS_DEV && !FIREBASE_PRIVATE_KEY;`